earth2mip/inference_medium_range.py

```diff
@@ -14,36 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import IO, List
 import logging
 
 import os
+import tempfile
+import pandas as pd
 import xarray as xr
 import torch
 import argparse
 import numpy as np
 import datetime
-import sys
 from earth2mip import config
 from earth2mip import time_loop, initial_conditions
 from earth2mip.initial_conditions import hdf5
 from earth2mip import _cli_utils
+import earth2mip.forecast_metrics_io
 from modulus.distributed.manager import DistributedManager
 
 
 __all__ = ["score_deterministic"]
 
 
-def get_times():
-    # the IFS data Jaideep downloaded only has 668 steps (up to end of november 2018)
-    nsteps = 668
-    times = [
-        datetime.datetime(2018, 1, 1) + k * datetime.timedelta(hours=12)
-        for k in range(nsteps)
-    ]
-    return times
-
-
 class RMSE:
```

Comment:
Why is this called RMSE when it just returns MSE?

Reply:
legacy. it used to always call .gather after scattering the .calls.